kafka-plugins-0.10/src/main/java/io/cdap/plugin/connector/KafkaConnector.java

```diff
@@ -58,7 +58,7 @@
 @Plugin(type = Connector.PLUGIN_TYPE)
 @Name(KafkaConnector.NAME)
 @Category("Messaging Systems")
-@Description("Kafka Connector to browse and sample topics using kafka 0.10.2 api")
+@Description("Connection to browse and sample data from Kafka topics.")
```

Comment:
Should keep connector here

Reply:
This is per request of our document writer : https://cdap.atlassian.net/browse/CDAP-18109